schema/annotation-source.yaml

```diff
@@ -96,7 +96,12 @@ $defs:
         description: MUST be "Agent".
       name:
         type: string
-
+     agent_type: 
```

Comment:
Figured this may be useful to be able to specify if the agent is a person, organization, or software agent.. VA has considered defining specific Agent subtypes for each of these - but is waiting specific uses cases to arise.

Reply:
Agent_type is fine. It's similar to what we are asking for at the contribution level above (re: activity type). So for consistency maybe we add a contribution_type to support the ability to capture the action, event or activity that the contribution reflects.

I'm good with this consistent approach

Reply:
I don't think contribution_type would be an enumeration. Probably a coding

Reply:
@larrybabb I made it an enum because the intent for me is to simply indicate the agent is a person, org, or software.  I gleaned from Alex's explanation that you guys use enums when the permissible values are meant to be a fixed set not to be extended or changed.  If you see the utility of this attribute differently, and would want to put other/more specific terms here, feel free to change it to a Coding.

Reply:
For contribution_subtype we will likely want a Coding as there may be a bunch of unknown activities that folks will want to identify as contributions.  I do not think we can control this vocabulary out of the gate.